scripts/add-dod-cas.sh

```diff
@@ -4,11 +4,20 @@
 
     # Location of bundle from DISA site
     bundle=https://dl.dod.cyber.mil/wp-content/uploads/pki-pke/zip/unclass-certificates_pkcs7_DoD.zip
-
+    # Copy the Checksum file
+    cp ./scripts/certificates_pkcs7_v5_11_dod.sha256 /usr/local/share/ca-certificates/
     # Extract the bundle
     cd /usr/local/share/ca-certificates
-    wget $bundle
+    wget --no-check-certificate $bundle
```

Comment:
Thought: when the certificate issue is resolved might be nice to just leave the sha check in anyway even if we can remove the `--no-check-certificate` flag.